app/models.py

```diff
@@ -199,8 +199,6 @@ class ServiceUser(BaseModel):
 
     __table_args__ = (UniqueConstraint("user_id", "service_id", name="uix_user_to_service"),)
 
-    user = db.relationship("User")
```

Comment:
Uncessary now with the SQLAlchemy upgrade? or it really removes the relationship, which then I wonder why the class is still named `ServiceUser`? 🤔

Reply:
Note that the table still has the user_id in there. 
This change removes the warning
https://docs.sqlalchemy.org/en/14/errors.html#error-qzyx
We could explore the use of `back_populates` but since it's only used once in the code it's easier to just eliminate this line